packages/vue/src/components/organisms/SfHeader/SfHeader.vue

```diff
@@ -144,6 +170,13 @@ export default {
       type: String,
       default: "",
     },
+    /**
+     * Navigation items
+     */
+    menuItems: {
+      type: [String, Array],
```

Comment:
`[String, Array]`? I think it should just be an array type as it's default value is an array ;)

Reply:
Ok, should be an array.